gitlab/const.py

```diff
@@ -17,6 +17,42 @@
 
 from gitlab.__version__ import __title__, __version__
 
+# NOTE(jlvillal): '__deprecated_all__' (our own custom name, not a Python
+# standard name) only affects users accessing constants via the top-level
+# namespace. See 'gitlab/__init__.py:__getattr__()' for the consumer of
+# '__deprecated_all__' For example 'x = gitlab.NO_ACCESS'.  We want users to
+# instead use constants by doing code like: gitlab.const.NO_ACCESS.
+__deprecated_all__ = [
```

Comment:
Hmm I guess my idea for using `__all__` was to also potentially control `from gitlab.const import *` but if done like this, we then wouldn't use it like that. So in that case it can even just be `_DEPRECATED` and not mix it with the builtin magic?

Reply:
Done. Changed to _DEPRECATED.

If you want to prohibit consumers from doing `from gitlab.const import *` then I can go back to `__all__`.

But I guess if they want to do it we can just let them. I know I despise `from blah import *` but I guess I think we should let people do that if they want to. But yuck if they do! 🙂